lib/msal-browser/test/cache/BrowserStorage.spec.ts

```diff
@@ -16,7 +16,7 @@ class TestCacheStorage implements ICacheStorage {
     getItem(key: string): string {
         throw new Error("Method not implemented.");
     }
-    removeItem(key: string): void {
+    removeItem(key: string): boolean {
```

Comment:
Why boolean?

Reply:
Would it be more helpful if an exception was thrown instead?

Reply:
We can revisit this at another date.